iris_markdownregression_EH.Rmd

```diff
@@ -0,0 +1,89 @@
+---
+title: "Homework 2: Funky Functions"
+author: "Ethan"
+date: "2020-01-22"
+output: html_document
+---
+
+Hello again! This time I would like to present my function that draws not only best fit line through a data set, but also illustrates why the line of best fit actually is the line of best fit.
+
+First, instead of calling the Iris dataset from within R, we will be reading it in from a ".csv" file: 
+```{r}
+iris <- read.csv("iris.csv")
+```
+
+Next, we will apply some standard housekeeping to make the data easier to work with:
+```{r}
+names(iris) <- c("sepalL", "sepalW",  "petalL", "petalW", "species")
+species <- iris$species
+color <- rep("red", 150)
```

Comment:
Why set it all as red if you're changing all 3 species? Is this to create a vector and then reassign the reds of the vector?

Reply:
That exactly what I wanted to do. I made a vector containing 150 colors first, then replaced each of the colors with the colors of my choosing. There is definitely a way to do this without the first step, I just haven't done that.